src/patternfly/components/Wizard/wizard.scss

```diff
@@ -131,6 +131,9 @@
   --pf-c-wizard__outer-wrap--BackgroundColor: var(--pf-global--BackgroundColor--100);
   --pf-c-wizard__outer-wrap--lg--PaddingLeft: var(--pf-c-wizard__nav--lg--Width);
 
+  // Main
+  --pf-c-wizard__main--ZIndex: var(--pf-global--ZIndex--xs);
```

Comment:
This z-index didn't seem to affect anything when testing locally, do we need it?

Reply:
We have the same thing on the page component's main section. I think it's useful just to make sure the main section has a lower z-index than the rest of the sections, so if someone adds something in the main section with `z-index: 999999999;`, it will only be relative to the main section and won't show above the header/nav.

Reply:
Ah, I see. In that case, `.pf-c-wizard__header` should also have a z-index.

Reply:
Yeah, we can, not sure if it's necessary. In the previous comment, I was combining the concepts of the page component (header/sidebar/main) and the wizard (toggle/nav/main) because of the stacking order needed for the layout, so it may have come across wrong. The z-index on main in the page component isn't necessarily to keep things from main displaying on top of the header (which isn't clear from what I said :D), but more for ensuring anything like a dropdown menu that originates from the header will always appear on top of whatever is in the main section. In the case of the wizard, we don't have that kind of need. I think the main concern in wizard is that we want to make sure nothing from main displays above the nav when it's expanded (on mobile). Then the toggle needs to display on top of nav because of the toggle's drop shadow, which is why it has the highest z-index.

Reply:
I read you. In the case that someone does something crazy with z-indexing and absolute positioning, say for a button, this might happen. It's the same concept as page component and an edge case for sure, so ¯\_(ツ)_/¯

![Screen Shot 2019-06-12 at 12 10 41 PM](https://user-images.githubusercontent.com/5385435/59368163-a2fa9a80-8d0b-11e9-9635-e62157d04b66.png)

Looks good to me either way 👍

Reply:
Ah gotcha! Agree that it's an edge case, but adding the same z-index that is on the toggle to the header would be more consistent with the page component and there is no downside that I can think of anyways, and it addresses this scenario. Good call.

Reply:
👍

Reply:
Updated @mcoker @mattnolting